CBLClient/Collection.py

```diff
@@ -0,0 +1,110 @@
+from CBLClient.Client import Client
```

Comment:
Could you run flake8 and run_repo_tests.sh, and check the review's boxes once they pass? 
It's easily done by running source setup.sh and then flake8 and sh run_repo_tests.sh

The Jenkins job fail if flake8 doesn't pass.

Reply:
./CBLClient/Database.py:398:1: W293 blank line contains whitespace
./CBLClient/Database.py:412:1: W293 blank line contains whitespace

will this be considered a flake8 error?
for repo_run_tests.sh there is no error.